code/_onclick/hud/screen_objects.dm

```diff
@@ -45,8 +45,7 @@
 
 /atom/movable/screen/Initialize(mapload, datum/hud/hud_owner)
 	. = ..()
-	if(hud_owner && istype(hud_owner))
-		hud = hud_owner
```

Comment:
is this the ONLY place we set hud in the codebase?

Reply:
from a glance, it seems so. since hud is also a private var